src/App.css

```diff
@@ -3,7 +3,7 @@
 }
 
 .App-header {
-    background-color: #282c34;
+    background-color: #282c34; /*antiquewhite*/
```

Comment:
In the future: This change is independent from the `checker` changes and should not be in this PR.

Rationale: PRs/merges are mean't to contain atomic changes to your project. The purpose is so they can be rolled back (unmerged) if there are any bugs. If you glue two independent changes together in a single PR then unmerges will unnecessarily unmerge working functionality.